integration-cli/docker_cli_network_unix_test.go

```diff
@@ -1128,10 +1128,10 @@ func (s *DockerNetworkSuite) TestDockerNetworkConnectDisconnectToStoppedContaine
 
 func (s *DockerNetworkSuite) TestDockerNetworkConnectPreferredIP(c *check.C) {
 	// create two networks
-	dockerCmd(c, "network", "create", "--subnet=172.28.0.0/16", "--subnet=2001:db8:1234::/64", "n0")
+	dockerCmd(c, "network", "create", "--ipv6", "--subnet=172.28.0.0/16", "--subnet=2001:db8:1234::/64", "n0")
```

Comment:
`--ipv6` was added to master for `docker network create` in line what we have for `docker daemon`, but libnetwork was not honoring it fully.
With this vendoring, we are filling the gap: If ipv6 stack is not explicitly enabled for the network, the user specified ipv6 subnet will be ignored.